map/framework.hpp

```diff
@@ -713,5 +713,11 @@ class Framework
   editor::UserStatsLoader m_userStatsLoader;
   //@}
 
+public:
+  //@{
```

Comment:
Please, don't use it unless you know what does it mean and why it was used here.